pyracf/common/security_admin.py

```diff
@@ -16,36 +16,9 @@ class SecurityAdmin:
     """Base Class for RACF Administration Interface."""
 
     _valid_segment_traits = {}
-    _common_base_traits_data_set_generic = {
-        "base:aclcnt": "racf:aclcnt",
-        "base:aclacnt": "racf:aclacnt",
-        "base:aclacs": "racf:aclacs",
-        "base:aclid": "racf:aclid",
-        "base:acl2cnt": "racf:acl2cnt",
-        "base:acl2acnt": "racf:acl2acnt",
-        "base:acl2acs": "racf:acl2acs",
-        "base:acl2cond": "racf:acl2cond",
-        "base:acl2ent": "racf:acl2ent",
-        "base:acl2id": "racf:acl2id",
-        "base:acsaltr": "racf:acsaltr",
-        "base:acscntl": "racf:acscntl",
-        "base:acsread": "racf:acsread",
-        "base:acsupdt": "racf:acsupdt",
-        "base:all": "racf:all",
-        "base:audaltr": "racf:audaltr",
-        "base:audcntl": "racf:audcntl",
-        "base:audnone": "racf:audnone",
-        "base:audread": "racf:audread",
-        "base:audupdt": "racf:audupdt",
-        "base:authuser": "racf:authuser",
-        "base:fvolume": "racf:fvolume",
-        "base:gaudaltr": "racf:gaudaltr",
-        "base:gaudcntl": "racf:gaudcntl",
-        "base:gaudnone": "racf:gaudnone",
-        "base:gaudread": "racf:gaudread",
-        "base:gaudupdt": "racf:gaudupdt",
-        "base:generic": "racf:generic",
-    }
+    _common_base_traits_data_set_generic = {}
+    __secret_traits = {}
```

Comment:
I don't think these need to be initialized here.

Reply:
I would try getting rid of these and see what happens.

Reply:
Nothing broke, I'll keep them out.